tests/adafruit_gps_test.py

```diff
@@ -467,3 +484,19 @@ def test_GPS_update_from_GSV_both_parts_sats_are_removed():
             assert gps.update()
             assert gps.satellites == 2
             assert set(gps.sats.keys()) == {"GP12", "GP14", "GP13", "GP15"}
+
+
+@pytest.mark.parametrize(
+    ("input_str", "exp", "neg"),
+    (
+        (["3723.2475", "N"], (37, 23.2475), "S"),
```

Comment:
The `neg` values should be lower case, I believe, based on the comparison in `_read_deg_mins()`.

Reply:
yeah, we use `.lower` inside `_read_deg_mins` so both will work. I think the NMEA-Standard has them upper case all the time. I'd suggest leaving it as is?

Reply:
I believe the provided `neg` parameter needs to already be lower case while the letter in the string is made lower case for the comparison.  Otherwise it'll compare a coerced lower case in the string to a provided upper case.

Reply:
okay - changed.